compiler/main.nim

```diff
@@ -315,6 +315,7 @@ proc mainCommand*(graph: ModuleGraph) =
 
       var dumpdata = %[
         (key: "version", val: %VersionAsString),
+        (key: "nimPath", val: %(getAppFilename())),
```

Comment:
That's not the "nimPath". Shouldn't the "nimPath" point to Nim's stdlib?

Reply:
that would be existing key `libpath` but I'm open to suggestions for another name;
nimExe?
nimBinary?

Reply:
ok i renamed to nimExe since we have `getCurrentCompilerExe`